src/NECompletion/TestTypingVisitor.class.st

```diff
@@ -54,8 +54,10 @@ TestTypingVisitor >> visitBlockNode: aBlockNode [
 ]
 
 { #category : #visiting }
-TestTypingVisitor >> visitGlobalNode: aGlobalNode [ 
-	aGlobalNode propertyAt: #type put: (Smalltalk globals at: aGlobalNode name) class.
+TestTypingVisitor >> visitGlobalNode: aGlobalNode [
+	aGlobalNode
+		propertyAt: #type
+		put: aGlobalNode binding assoc read class
```

Comment:
Maybe we could have a method on global node to avoid hurting Demeter too much in a next PR .

Reply:
yes, I make a note in the backlog